package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "crud",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "parcel ./src/index.html",
+    "build": "rm -rf dist; parcel build --detailed-report",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fmt": "prettier --write .",
+    "fix": "prettier --write src/; eslint --fix src/",
```

The `src/` thing was only to avoid `fix` formatting the ginormous `data/` folder in parking lot map.

```suggestion
    "fix": "prettier --write .; eslint --fix .",
```